spec/unit/actions/route_update_spec.rb

```diff
@@ -41,7 +41,8 @@ module VCAP::CloudController
     end
 
     let(:message) { RouteUpdateMessage.new(body) }
-    let(:route) { Route.make }
+    let(:space) { Space.make }
+    let(:route) { Route.make(space:) }
 
     subject { RouteUpdate.new }
     describe '#update metadata' do
```

Comment:
For the `describe '#update metadata'` block, you should add this:
```
before do
  expect(ProcessRouteHandler).not_to receive(:new)
end
```

This will also force you to change the implementation, as I would expect that Diego should only be notified if some route option changed.